src/app/tools/ink_processing.h

```diff
@@ -1240,10 +1260,17 @@ class BrushInkProcessingBase : public DoubleInkProcessing<BrushInkProcessingBase
     return true;
   }
 
+  inline const int brushWidth() const {
+    return m_isBoundsRotated ? m_height : m_width;
+  }
+  inline const int brushHeight() const {
+    return m_isBoundsRotated ? m_width : m_height;
+  }
```

Comment:
Remove `const` in return type:
```suggestion
  inline int brushWidth() const {
    return m_isBoundsRotated ? m_height : m_width;
  }
  inline int brushHeight() const {
    return m_isBoundsRotated ? m_width : m_height;
  }
```